examples/pandas/split-apply-combine/my_full_dag

```diff
@@ -0,0 +1,51 @@
+// Dependency Graph
```

Comment:
we usually don't commit these files.

Reply:
Should we add a global rule in `.gitignore` ?

Reply:
I hadn't thought of that, but unfortunately not sure what rule we'd place since the file doesn't have an extension... 🤔

Reply:
I added a local `.gitignore` for now